openvdb_points/tools/PointDataGrid.h

```diff
@@ -439,12 +471,17 @@ class PointDataLeafNode : public tree::LeafNode<T, Log2Dim> {
         return totalPointCount;
     }
 
-    void deactivateEmptyVoxels()
+    /// @brief Uses the point count of each voxel to update the active voxel mask on this
+    /// leaf node. If @a deactivateEmptyVoxels is enabled, then voxels with a point count
+    /// of zero will be deactivated. If @a activateNonEmptyVoxels is enabled, then voxels
+    /// with a non-zero point count will be activated. By default, both of these are enabled.
+    void updateActiveVoxelMask(bool deactivateEmptyVoxels=true, bool activateNonEmptyVoxels=true)
```

Comment:
That's fair enough - I'd prefer to just keep things simple and omit these arguments. Will address shortly...